README.md

```diff
@@ -169,6 +169,11 @@ If set to True, escape special chars in userdn when authenticating in LDAP.
 On some LDAP servers, when userdn contains chars like '(', ')', '\' authentication may fail when those chars
 are not escaped.
 
+#### `LDAPAuthenticator.user_info_attributes` ####
+
+An optional list of attributes to be fetched for a user after login.
+If found these will be returned as `auth_state`.
```

Comment:
Just bike-shedding the name but if it's a list of attributes to return as the `auth_state` dictionary perhaps `auth_state_attributes` would be more descriptive/discoverable